src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_json_debug_options.py

```diff
@@ -61,6 +63,9 @@ def update_fom_debug_options(self, debug_options):
 
         # Note: _max_exception_stack_frames cannot be set by debug options.
 
+        if 'ENABLE_GUI_EVENT_LOOP' in debug_options:
+            self.enable_gui_event_loop = debug_options.get('ENABLE_GUI_EVENT_LOOP')
+
```

Comment:
This bit shouldn't be necessary - `debugOptions` is a legacy property for boolean flags in debug config that we had before deciding to just use separate boolean properties; it is really only kept for backwards compatibility.

Reply:
I was wondering what those optoins are for... Removed in https://github.com/microsoft/debugpy/pull/787/commits/f07013df3b0e1932f2f6701e8d8b5dbe9bb2c24a